pkg/compiler/fuzz/fuzz.go

```diff
@@ -0,0 +1,14 @@
+package compiler_test
+
+import (
+	"github.com/MontFerret/ferret/pkg/compiler"
+)
+
+func Fuzz(data []byte) int {
+	c := compiler.New()
+	p, err := c.Compile(string(data))
+	if err!=nil || p==nil{
```

Comment:
variables and operators should be separated by space. So, line should be `if err != nil || p == nil`